data/json/items/ranged.json

```diff
@@ -138,9 +138,9 @@
     "type": "GUN",
     "symbol": "(",
     "color": "green",
-    "name": "M72 LAW",
-    "name_plural": "M72 LAW",
-    "description": "A disposable light anti-tank rocket launcher.  Once fired, it cannot be reloaded and must be disposed of.",
+    "name": "disposible rocket launcher",
```

Comment:
Seems too generic to me. Different disposable rocket launchers may become available at some point (or even are already available), which would lead to confusion.

Reply:
I agree that it wouldn't be a good idea to have two different items with the same name, but I don't see what would be confusing about such a name as long as it describes only one thing.